### DISCUSSION TITLE
FreshRSS is Wonderful Software

### DISCUSSION THREAD

**Discussion by nattaylor** [FreshRSS/FreshRSS#3414] [General]
I'm a long time FreshRSS user and was always frustrated that when reading a long article, then advancing to a short one would leave me with a blank screen.  After a bit of grumbling, lo and behold, there is configurable setting for "Stick the article to the top when opened."  When the authors have designed configurable solutions to many problems, it is a sign of delightful software.  FreshRSS is a delight to use!  Thank you!

**Comment by Alkarex** [comment]
Thanks for those kind words :-)
